### PR TITLE
chore(*): update-readme-16-jun-2023

### LINKED ISSUE #73: Update of the README
Update the README documentation

### PR DESCRIPTION
**Pull request by radomirchev**
closes #73 
Update of the Readme

### CODE REVIEW

**Comment by radomirchev** [comment]
@ChronosSF @damyanpetev ready for review

**Review by kdinev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,93 @@
...
+
+## Browser Support
+
+![chrome_48x48](https://user-images.githubusercontent.com/2188411/168109445-fbd7b217-35f9-44d1-8002-1eb97e39cdc6.png) | ![firefox_48x48](https://user-images.githubusercontent.com/2188411/168109465-e46305ee-f69f-4fa5-8f4a-14876f7fd3ca.png) | ![edge_48x48](https://user-images.githubusercontent.com/2188411/168109472-a730f8c0-3822-4ae6-9f54-785a66695245.png) | ![opera_48x48](https://user-images.githubusercontent.com/2188411/168109520-b6865a6c-b69f-44a4-9948-748d8afd687c.png) | ![safari_48x48](https://user-images.githubusercontent.com/2188411/168109527-6c58f2cf-7386-4b97-98b1-cfe0ab4e8626.png)
+--- | --- | --- | --- | --- |
+Latest ✔ | Latest ✔ | Latest ✔ | Latest ✔ | Latest ✔ |
+
+## Purpose
+
+This repository is intended for clients to create feature requests, log bugs, and ask questions related to [Ignite UI for Blazor](https://www.infragistics.com/products/ignite-ui-blazor). Please note that this repository does not contain any source code or allow for direct code changes.
```
Comment (kdinev) [inline README.md:23]:
```suggestion
This repository is intended for users to create feature requests, submit issues, and ask questions related to [Ignite UI for Blazor](https://www.infragistics.com/products/ignite-ui-blazor). Please note that this repository does not contain any source code or allow for direct code changes.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+
+## Overview
+
+|Components|Status|Documentation|
+|:--|:--:|:--:|
+|Combo|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/inputs/combo/overview)
```
Comment (kdinev) [inline README.md:29]: Order them alphabetically.

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+|Radio|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/inputs/radio)|
+|Radio Group|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/inputs/radio)|
+|Ripple|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/inputs/ripple)|
+|Switch|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/inputs/switch)|
+
+## Grid components
+
+|Components|Status|Documentation|Released Version|
+|:--|:--:|:--:|:--:|
+|Pivot Grid|:white_check_mark:|[Docs](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/grids/pivot-grid/overview)|
```
Comment (kdinev) [inline README.md:69]: Order them alphabetically.

File: README.md [hunk 4]
```diff
@@ -1 +1,93 @@
...
+
+## Usage
+
+### [Ignite UI for Blazor Server App](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/general-getting-started#install-ignite-ui-for-blazor)
+
+### [Ignite UI for Blazor WebAssembly](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/general-getting-started-blazor-client)
+
+### [Dock Manager - EXCLUSIVE FEATURE](https://www.infragistics.com/products/ignite-ui-blazor/blazor/components/layouts/dock-manager)
+
+Provide a complete windowing experience, splitting complex layouts into smaller, easier-to-manage panes.
```
Comment (kdinev) [inline README.md:81]:
```suggestion
Provide a complete multi-window experience, splitting complex layouts into smaller, easier-to-manage, and modify panes. 
```

**Review by kdinev — CHANGES_REQUESTED** [review]

**Review by kdinev — APPROVED** [review]

**Pull request merged**